src/components/Type/Type.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import './Type.css';
+import {Link} from 'react-router-dom';
 
-const Type = ({img, title}) => {
+const Type = ({img, title, category}) => {
     return (
-      <article className="productType" style={{backgroundImage:`url(${img})`}}>
-        <h3>{title}</h3>
-      </article>
+      <Link to={`/${category}/${title}`}>
```

Comment:
This interpolation has helped me see how we might be able to try up our category components.